app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def move_file(command: str) -> None:
+    file_name, new_path = command.split(" ")[1:]
+
+    with open(file_name, "r") as sorce:
+        text = sorce.read()
```

Comment:
The file is opened and read, but there is no error handling for cases where the file might not exist. Consider adding a try-except block to handle potential `FileNotFoundError` exceptions.